server/nginx.conf

```diff
@@ -0,0 +1,38 @@
+events {
+    worker_connections  1024;
+}
+daemon off;
+http {
+    server {
+        server_name localhost;
+        listen 7878;
+
+        rewrite ^/(.*) /$1 break;
+        proxy_ignore_client_abort on;
+        proxy_set_header  X-Real-IP  $remote_addr;
+        proxy_set_header  X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header  Host $http_host;
+
+        # Enable CORS requests
+        proxy_set_header Access-Control-Allow-Origin "*";
+
+        location ~ ^(/|/query)$ {
+            # limit_except GET POST OPTIONS {
+            #         deny  all;
+            # }
+            proxy_pass http://oxigraph:7878;
+            proxy_pass_request_headers on;
+        }
+
+        location /update {
```

Comment:
You also might want to expose the `/store` path providing [the SPARQL graph store protocol](https://www.w3.org/TR/sparql11-http-rdf-update/)/